readme.md

```diff
@@ -356,9 +356,9 @@ Thanks for contributing! 🦋🙌
 - [](# "useful-not-found-page") 🔥 [Adds possible related pages and alternatives on 404 pages.](https://user-images.githubusercontent.com/1402241/46402857-7bdada80-c733-11e8-91a1-856573078ff5.png)
 - [](# "selection-in-new-tab") [Adds a keyboard shortcut to open selection in new tab when navigating via <kbd>j</kbd> and <kbd>k</kbd>: <kbd>shift</kbd> <kbd>o</kbd>.](https://github.com/refined-github/refined-github/issues/1110)
 - [](# "close-out-of-view-modals") [Automatically closes dropdown menus when they’re no longer visible.](https://user-images.githubusercontent.com/1402241/37022353-531c676e-2155-11e8-96cc-80d934bb22e0.gif)
-- [](# "parse-backticks") [Renders `` `text in backticks` `` in issue titles, commit titles, and more places.](https://user-images.githubusercontent.com/170270/55060505-31179b00-50a4-11e9-99a9-c3691ba38d66.png)
+- [](# "parse-backticks") [Renders `` `text in backticks` `` in issue titles, commit titles, and more places.](https://github-production-user-asset-6210df.s3.amazonaws.com/140871606/257086163-f3064ad1-e344-4ce0-a084-7e59fde5dd36.png)
```

Comment:
This screenshot is no longer correct. This feature now only complements GitHub'a native formatting because they forgot to apply this to some parts of the site. The highlighted parts here are natively formatted, for example.